### ISSUE TITLE
Not an issue but rather a slightly related Question

### ISSUE THREAD

**Issue by batboyshark** [riderkick/FMD#1470]
Hey, i like this program i think it is very good for manga though it has a few bugs.

I would like to know if it is possible for the makers to make a similar program for american comics if its possible i would like to request it thank you very much.

**Comment by 1aam2am1** [comment]
No, but there are many sites that you can download from. I don't know if yous site is implemented but you can check it in Options => Websites => Websites => English => here is the list of implemented sites => check the one that you need.

**Comment by kevin01523** [comment]
i think there is an american comics site supported though its an affiliate of kissmanga and kissanime
